lib/p2p/Pool.ts

```diff
@@ -54,6 +54,13 @@ class Pool extends EventEmitter {
       return;
     }
 
+    // TODO differentiate between testnet & mainnet
+    this.logger.info('Connecting to Seed Peers');
+    this.addOutbound(new SocketAddress('xud1.test.exchangeunion.com', 8885));
+    this.addOutbound(new SocketAddress('xud2.test.exchangeunion.com', 8885));
+    this.addOutbound(new SocketAddress('xud3.test.exchangeunion.com', 8885));
```

Comment:
I don't think this should be placed inside of the code.

My preferred approach would be to simply start working with our `db:init` task, and to make it more suitable for production use, so that after setting up `xud`, the database will be initialized with initial values for `hosts` table, as well as for other tables.

If for some reason we'll want to separate the seed list from the `hosts` table, we can use the common approach of having a separate file for holding that data.

Reply:
This is how bitcoin does it: https://github.com/bitcoin/bitcoin/blob/master/src/chainparams.cpp#L128

Reply:
That's Bitcoin Core. Other implementations usually implement it as a param, for the obvious reasons. But that's nothing major. 
In our case, we have a separate RDBMS, and we manage host identities, so I think we should just use that, unless we can think of a reason for not doing so.

Reply:
Ok.